package.json

```diff
@@ -28,19 +28,20 @@
   "devDependencies": {
     "bistre": "^1.0.1",
     "browserify": "^14.3.0",
-    "budo": "^9.4.0",
+    "budo": "^11.8.4",
     "uglify-js": "^2.8.0"
   },
   "dependencies": {
+    "corslite": "^0.0.7",
```

Comment:
Is this new dependency really necessary?

Reply:
I don't know. I think I arrived at this via npm audit. And previously I always had `npm install corslite` as an additional step, without it would never build. But I have very little experience with npm and javascript in general, so I can just remove it if you feel that would be better.